test/atm/TLRTokenRevokeVestingTest.js

```diff
@@ -75,7 +75,7 @@ contract('TLRTokenRevokeVestingTest', function (accounts) {
 
             try {
                 // Invocation
-                const result = await instance.revokeVesting(receipent, 0, { from: sender });
+                const result = await instance.revokeVesting(receipent, 1, { from: sender });
```

Comment:
Just making sure that this is a valid change since it doesn't look like much was updated surrounding this function.

Reply:
Yes, it's due to the vestingId update